.gitattributes

```diff
@@ -1 +1,2 @@
 * text eol=lf
+* binary text=auto
```

Comment:
I don't understand what this means? Binary text? You can remove this line.

You should generally specify which files are binary, and which files are text manually, if git has problems with them. Otherwise, binary files can be corrupted.

For the most part, git is smart enough to know which is which, you may add manually files without extensions like gradlew, or other exceptions that git doesn't handle well (you probably noticed some of them already).

https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings

This guide has an example. You can add file extensions, or individual files, and set them to binary.

Reply:
changed it to `* binary` which does nothing for binary files

Reply:
Yeah, I don't think that line does what we want to do. It sets every file to binary, which is wrong. Setting text to auto should automatically ignore binaries.

And if there are some binaries that git is not smart enough to recognize, you can manually add them. See the example above.